package.json

```diff
@@ -9,18 +9,21 @@
     "enabledApps": "cowswap-frontend cowswap-frontend-e2e abis ui ui-utils widget-lib widget-react"
   },
   "scripts": {
-    "start": "nx run cowswap-frontend:serve",
+    "start": "yarn start:cowswap",
+    "start:cowswap": "nx run cowswap-frontend:serve",
     "start:widget": "nx run widget-configurator:serve",
-    "build": "yarn run build-app",
-    "build-app": "cross-env NODE_OPTIONS=--max-old-space-size=32768 nx build cowswap-frontend",
-    "build-widget": "nx build widget-configurator",
-    "ci:build": "yarn run build-app && yarn run build-widget",
+    "start:cosmos": "nx run cowswap-frontend:cosmos:run",
+    "build": "yarn run build:cowswap",
+    "build:all": "nx run-many -t build",
+    "build:cowswap": "cross-env NODE_OPTIONS=--max-old-space-size=32768 nx build cowswap-frontend",
+    "build:widget": "nx build widget-configurator",
+    "build:cosmos": "cross-env NODE_OPTIONS=--max-old-space-size=32768 nx run cowswap-frontend:build:cosmos",
```

Comment:
I think we no longer need `--max-old-space`, no?

Reply:
i don't know. I did a PR in isolation so we can test https://github.com/cowprotocol/cowswap/pull/3311